### DISCUSSION TITLE
XMR Node available: False

### DISCUSSION THREAD

**Discussion by j2JHHire** [btcpayserver/btcpayserver#5141] [Technical Support] [answered]
![Screenshot 2023-07-02 at 12-16-02 Monero Settings](https://github.com/btcpayserver/btcpayserver/assets/138367532/59ac0188-664b-492e-bdea-f0c1ea1dc2a0)

```
~/BTCPayServer/btcpayserver-docker# docker exec -it btcpayserver_monerod monero-wallet-cli
Error response from daemon: Container ***** is restarting, wait until the container is running
```

Issue with Monero. No node and creating a wallet is not possible either. Have this problem for the first time. Can anyone help?

**Comment by xrybxby** [comment]
ive had this issue for about a week, i dont know when or how its getting fixed, its not a priority because its altcoin build unfortunately for me its my main revenue source so I dont know what to do. Just gotta be patient

**Reply by j2JHHire** [reply]
So there is no solution for monero in the near future?

**Comment by Kukks** [comment, marked as answer]
Try updating.

**Reply by j2JHHire** [reply]
Ok. Now it works :)

**Comment by Delitants** [comment]
Updating doesn't help, problem is chronic.
